typings/index.d.ts

```diff
@@ -2604,7 +2604,7 @@ declare module 'discord.js' {
     channelCreate: [channel: GuildChannel];
     channelDelete: [channel: DMChannel | GuildChannel];
     channelPinsUpdate: [channel: Channel | PartialDMChannel, date: Date];
```

Comment:
```suggestion
    channelPinsUpdate: [channel: TextChannel | NewsChannel | PartialDMChannel, date: Date];
```
While we are at it, I found some more typing issues of the same nature. This is one of them, the other one is the `typingStart` event but github won't let me pick that.

Reply:
superseded by internal discussion applied in 8143f56 769489a

Reply:
I see. btw the jsdoc description for these two events are a little inconsistent. Like for `channelPinsUpdate` it says `DMChannel` or `TextChannel` while for `typingStart` it is generic `channel`. As both of these events have same typings they should have same jsdoc too